### PR TITLE
Create 2017-04-8-web ext.md

### PR DESCRIPTION
**Pull request by Aishwarya5**
A Big Web-extension Hackathon to be held on April -8

### CODE REVIEW

File: docs/_posts/2017-04-8-web ext.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+
+#event information
+title:  "web extesion event invitation"
+date:   2017-04-08
+start_time: "9:30"
+end_time: "5:00"
```
Comment (iamVP7) [inline docs/_posts/2017-04-8-web ext.md]: it should be in 24 hours format. and end time is 6:00 PM

File: docs/_posts/2017-04-8-web ext.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+#event organiser details
+
+chief guests: "Mr.Vishwaprasath & Mr.karthikayan"
+
+categories: "event"
+---
+
+   <p>The grandiose opening of Mozilla campus club which is eminently named as SKCETZILLA is planned to be held on April-8 ,which would be a great overture to all the up coming engineers of SKCET who wish to spend ample amount of their time to get into open source contributions and grow themselves with excellent skills.
```
Comment (iamVP7) [inline docs/_posts/2017-04-8-web ext.md]: Why using <p> tag

**Review by prasanthp96 — COMMENTED** [review]
file name should not have space "2017-04-8-web ext.md" should be "2017-04-08-web-ext.md"

File: docs/_posts/2017-04-8-web ext.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ layout: post
 title:  "web extesion event invitation"
 date:   2017-04-08
 start_time: "9:30"
-end_time: "5:00"
+end_time: "6:00"
```
Comment (iamVP7) [inline docs/_posts/2017-04-8-web ext.md]: use 24 hours time format.

**Pull request merged**